Features/CaseSearch/test_cases/conftest.py

```diff
@@ -10,7 +10,7 @@
 
 
 @pytest.fixture(scope="session")
-def environment_settings_casesearch():
+def environment_settings_multiselect():
```

Comment:
I am a bit confused here. Why is this renamed as multiselect but the one under multiselect is named as case search?

Reply:
Updated in https://github.com/dimagi/dimagi-qa/commit/0634ed543e3e5846a8be5e8b7292a2e6a8f513b2